src/MSBuild.Abstractions/MSBuildConversionWorkspace.cs

```diff
@@ -329,6 +329,13 @@ private bool IsSupportedProjectType(IProjectRootElement root, bool forceWeb)
                 Console.WriteLine($"{root.FullPath} contains an App.config file. App.config is replaced by appsettings.json in .NET Core. You will need to delete App.config and migrate to appsettings.json if it's applicable to your project.");
             }
 
+            if (!keepCurrentTFMs
+                && root.PropertyGroups.Any(pg => pg.Properties.Any(ProjectPropertyHelpers.IsVisualBasicProject))
+                && root.ItemGroups.Any(ig => ig.Items.Any(ProjectItemHelpers.IsReferencingSettingsSingleFileGenerator)))
+            {
+                Console.WriteLine($"{root.FullPath} uses code generators which will not be handled by try-convert. You can edit your vbproj to add support or remove these dependencies.");
+            }
```

Comment:
You don't return here. Just to confirm, you intend to give the warning message in this case but still allow try-convert to execute?

Reply:
Correct, the intention is to allow the user to proceed with a warning that we expect a compile error after the conversion. I believe this is similar in experience to how we handle web projects.